lib/ts/recipe/emailpassword/recipe.ts

```diff
@@ -53,24 +55,23 @@ export default class Recipe extends RecipeModule {
 
     isInServerlessEnv: boolean;
 
+    emailDelivery: EmailDeliveryIngredient<TypeEmailPasswordEmailDeliveryInput>;
```

Comment:
All recipes should have a way for the user to send emails themselves as well (in case they disable one of our APIs and wanna make their own). So something like:
```
EmailPassword.sendEmail(TypeEmailPasswordEmailDeliveryInput)
```

(same goes for sendSMS)

Reply:
Please make sure this is done properly for all recipes